service/src/main/kotlin/fi/espoo/evaka/shared/security/Action.kt

```diff
@@ -1979,11 +1980,13 @@ sealed interface Action {
     ) : ScopedAction<ServiceApplicationId> {
         ACCEPT(
             HasGlobalRole(ADMIN),
-            HasUnitRole(UNIT_SUPERVISOR).inPlacementUnitOfChildOfServiceApplication(),
+            HasUnitRole(UNIT_SUPERVISOR)
+                .inPlacementUnitOfChildOfServiceApplicationWithPilotFeature(),
         ),
         REJECT(
             HasGlobalRole(ADMIN),
-            HasUnitRole(UNIT_SUPERVISOR).inPlacementUnitOfChildOfServiceApplication(),
+            HasUnitRole(UNIT_SUPERVISOR)
+                .inPlacementUnitOfChildOfServiceApplicationWithPilotFeature(),
```

Comment:
Eikös tärkeämpää olisi tuoda toimintojen avauksien vaikutus kuntalaisen puolelle? Jos/kun feature flag poistetaan, kuka tahansa kuntalainen saa tehdä näitä hakemuksia, mutta yksiköissä niitä ei näe.

Reply:
ChildControllerCitizen.kt
```
                    val serviceApplicationPossible =
                        tx.getChildrenWithServiceApplicationPossibleOnSomeDate(
                            childIds.toSet(),
                            clock.today(),
                        )
                    val permittedActions =
                        accessControl
                            .getPermittedActions<ChildId, Action.Citizen.Child>(
                                tx,
                                user,
                                clock,
                                childIds,
                            )
                            .mapValues { (childId, actions) ->
                                actions.letIf(!serviceApplicationPossible.contains(childId)) {
                                    it - Action.Citizen.Child.CREATE_SERVICE_APPLICATION
                                }
                            }
                    children.map { c ->
                        ChildAndPermittedActions.fromChild(c, permittedActions[c.id]!!)
                    }
```

Reply:
Ai niin, se oli tämä.

Reply:
reverttasin vielä mm. ton `inPlacementUnitOfChildOfServiceApplicationWithPilotFeature` kun löysin et on jo olemassa `.withUnitFeatures(PilotFeature.SERVICE_APPLICATIONS)` jonka voi ketjuttaa mukaan acl-sääntöön